tasks/installation.yml

```
@@ -88,7 +92,7 @@
 
 - name: install modules from github
   git: # noqa 401
```

The `noqa 401` tag disabled the "Git checkouts must contain explicit version" warning. Since a version is provided now, we should be able to remove this.

```suggestion
  ansible.builtin.git:
```